snafu/benchmarks/dnsperf/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM registry.centos.org/centos:8
+
+COPY snafu/image_resources/centos8-appstream.repo /etc/yum.repos.d/centos8-appstream.repo
+
+# snafu dependencies
+## download, compile python 3.7
+RUN dnf install -y \
+        bzip2-devel \
+        gcc \
+        libffi-devel \
+        make \
+        openssl-devel \
+        xz-devel \
+        zlib-devel && \
+    curl https://www.python.org/ftp/python/3.7.7/Python-3.7.7.tgz -o /opt/Python-3.7.7.tgz && \
+    tar xzf /opt/Python-3.7.7.tgz --directory /opt
+
+RUN dnf install -y \
```

Comment:
Why are we installing redis and the ocp client? Is that for benchmark-operator?

Reply:
If you're not using them, then yes.